src/Text/Layout/Table/Cell.hs

```diff
@@ -46,6 +74,11 @@ instance (Cell a, Cell b) => Cell (Either a b) where
     dropLeft n = bimap (dropLeft n) (dropLeft n)
     dropRight n = bimap (dropRight n) (dropRight n)
     dropBoth l r = bimap (dropBoth l r) (dropBoth l r)
+    dropLeftNoPad n = either (second Left . dropLeftNoPad n) (second Right . dropLeftNoPad n)
+    dropRightNoPad n = either (first Left . dropRightNoPad n) (first Right . dropRightNoPad n)
+    dropBothNoPad l r = either (middle Left . dropBothNoPad l r) (middle Right . dropBothNoPad l r)
+      where
+        middle f (a, b, c) = (a, f b, c)
```

Comment:
This almost begs for introducing new types.  Then you could automatically derive `Functor`.  I am not completely opposed to the idea with tuples though.

Reply:
Yes, that's a good thought. I'll look into that over the next few days. If you want to merge #37 without those changes go ahead and I'll submit a separate PR later.

Reply:
I have introduced a new type for padding. Not only is it a functor, but it is a monad! This really simplifies the above code by allowing us to use `bitraverse`.